File: problems/daily/Java/1512. Number of Good Pairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int res = 0;
        int l = nums.length;
        for(int i = 0; i < l- 1;i++){
            for(int k = i + 1;k < l; k++){
                if(nums[i] == nums[k]) res++;
            }
        }
        return res;
    }
}

File: problems/daily/Python3/1512. Number of Good Pairs.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        ans = 0
        l = len(nums)
        for i in range(l):
            for j in range(i+1,l):
                if nums[i]== nums[j]:
                    ans+=1
        return ans